apps/lead/schema.py

```diff
@@ -480,6 +506,13 @@ class Query:
             page_size_query_param='pageSize'
         )
     )
+    lead_preview_attachments = DjangoPaginatedListObjectField(
+        LeadPreviewAttachmentListType,
+        pagination=PageGraphqlPagination(
+            page_size_query_param='pageSize',
+        )
+    )
+
```

Comment:
We need custom resolver as well

```suggestion
    @staticmethod
    def resolve_lead_preview_attachments(root, info, **kwargs) -> QuerySet:
        return get_lead_preview_attachment_qs(info)
```